docs/use_cases/met_tool_wrapper/PlotDataPlane/PlotDataPlane_netcdf.py

```diff
@@ -0,0 +1,115 @@
+"""
+PlotDataPlane: GRIB1 Input
```

Comment:
This line appears to be incorrect. I noticed that in the online documentation, the PlotDataPlane_grib1 example appears twice, instead of 1 for Grib, and one for netCDF. This will also require a change on line 5